### DISCUSSION TITLE
3/22(수) view의 생명주기

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#25] [General]
전에 타이머 만들기 했었죠? 거기서 viewDidLoad를 사용했을 텐데 viewDidLoad가 무엇인지 언제 사용해야하는지 viewDidLoad 말고 다른 생명주기는 무엇인지 공부해오시면 될 것 같습니다~~~ 🤓

**Comment by Eunho0922** [comment]
넹
